src/Framework/MSBuildEventSource.cs

```diff
@@ -0,0 +1,465 @@
+﻿using System;
+using System.IO;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+
+
+//
+// This captures information of how various key methods of building with MSBuild ran.
```

Comment:
We don't generally comment namespaces. If you want to, please use the XML doc comment syntax.